main.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express')
+// Get the client
+const bodyParser = require('body-parser');
+const { getLastIPAddress, insertLastIPAddress } = require('./db');
+const { getDistanceFromLatLonInKm, calculateRiskScore } = require('./utils');
+const Reader = require('@maxmind/geoip2-node').Reader;
+// Typescript:
+// import { Reader } from '@maxmind/geoip2-node';
+
+const options = {
+  // you can use options like `cache` or `watchForUpdates`
+};
+
+function getLatLong(IPAddress) {
+  return new Promise((resolve, reject) => {
+    Reader.open('./GeoLite2-City.mmdb').then(reader => {
+      const response = reader.city(IPAddress);
+      resolve({
+        latitude: response.location.latitude,
+        longitude: response.location.longitude
+      })
+    }).catch((error) => {
+      reject(error);
+    });
+  })
+}
+
+
+const app = express()
+const port = 3000
+
+app.use( bodyParser.json() );       // to support JSON-encoded bodies
+app.use(bodyParser.urlencoded({     // to support URL-encoded bodies
+  extended: true
+})); 
+// Get the client
+
+app.get('/', (req, res) => {
+  console.log("request recieved ", req);
+  res.send('Hello World!')
+})
```

Comment:
why does this API exist?

Reply:
Had planned initially to do it but deprioritised it for getting the service up and running.